packages/api/models/granules.js

```diff
@@ -353,10 +353,11 @@ class Granule extends Manager {
    * @returns {Promise}
    */
   async _validateAndStoreGranuleRecord(granuleRecord) {
+    const clonedGranuleRecord = cloneDeep(granuleRecord);
```

Comment:
Could we add a note here about why we're cloning it? Or change `recordIsValid` to not mutate (that seems way out of scope).

Reply:
A note seems fine.    Refactoring to not use ajv .... is most definitely out of scope.

Reply:
893273ca70

Reply:
Thanks. This is sort of unexpected behavior so I think it's good to have a note.